extensions/trace_utils/src/test/java/io/opentelemetry/extensions/trace/CurrentSpanUtilsTest.java

```diff
@@ -60,7 +61,6 @@ void withSpanRunnable() {
           assertThat(getCurrentSpan()).isSameAs(span);
         };
     CurrentSpanUtils.withSpan(span, false, runnable).run();
-    verifyNoInteractions(span);
```

Comment:
I think in this case we should use verifyNoFurtherInteractions and assert on the call to storeInContext to not lose the original assertion's intent.

Reply:
Thanks - I was actually a bit confused on the original intent but took another look. I went with explicitly checking end isn't called (as specified by the `false` above) which is symmetric with the other tests, I think this would have been clearer to me from the start. Hope this looks OK.